libenkf/applications/ert_tui/enkf_tui_analysis.c

```diff
@@ -72,7 +72,8 @@ void enkf_tui_analysis_select_module__(void * arg) {
   menu_type * menu = arg_pack_iget_ptr( arg_pack , 1 );
   
   {
-    analysis_config_type * analysis_config = enkf_main_get_analysis_config( enkf_main );
+    // This is really a const return.
+    analysis_config_type * analysis_config = (analysis_config_type *) enkf_main_get_analysis_config( enkf_main );
```

Comment:
I find this to be a bad idea! We should not silence this warning. I would prefer that we make `enkf_main_get_analysis_config` not return a `const` until `analysis_config` some time in the future actually is a const (if this ever happens).